code/modules/research/bepis.dm

```diff
@@ -2,11 +2,11 @@
 
 //economics defined values, subject to change should anything be too high or low in practice.
 
-#define MACHINE_OPERATION 100000
-#define MACHINE_OVERLOAD 500000
-#define MAJOR_THRESHOLD 3000
-#define MINOR_THRESHOLD 2000
-#define STANDARD_DEVIATION 1000
+#define MACHINE_OPERATION 400*CARGO_CRATE_VALUE
+#define MACHINE_OVERLOAD 2000*CARGO_CRATE_VALUE
```

Comment:
These should be unchanged as they're the power cost of the BEPIS, not credit values.